File: Practica_11/Python/Fila_A/Obra.py
from Anuncio import Anuncio
from Artista import Artista

class Obra:
    def __init__(self, titulox, materialx, a1x, a2x, bx):
        self.titulo = titulox
        self.material = materialx
        self.a1 = Artista("Pablo Picasso", 12345, 20)
        self.a2 = Artista("Claude Monet", 67890, 24)
        self.b = Anuncio(1, 3500)
        self.a1 = a1x
        self.a2 = a2x
        self.b = bx

    def mostrar(self):
        print("OBRA")
        print("Título:", self.titulo)
        print("Material:", self.material)
        self.a1.mostrar()
        self.a2.mostrar()
        if self.b is not None:
            self.b.mostrar()
        else:
            print("Sin anuncio.")

File: Practica_11/src/Java/Fila_A/Obra.java
package Java.Fila_A;

public class Obra {
	String titulo;
	String material;
	Artista a1 = new Artista("Pablo Picasso", 12345, 20);
	Artista a2 = new Artista("Claude Monet", 67890, 24);
	Anuncio b = new Anuncio(1, 3500);
	
	public Obra(String titulox, String materialx, Artista a1x, Artista a2x, Anuncio bx) {
		this.titulo=titulox;
		this.material=titulox;
		this.a1=a1x;
		this.a2=a2x;
		this.b=bx;
	}
	public void mostrar() {
		System.out.println("OBRA");
		System.out.println("Titulo: "+this.titulo);
		System.out.println("Material: "+this.material);
		a1.mostrar();
		a2.mostrar();
		b.mostrar();
	}
}
